csharp-basics/exercises/ClassesAndObjects/BankAccount/BankAccount.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BankAccount
+{
+    class BankAccount
+    {
+        private string _name;
+        private decimal _balance;
+
+        public BankAccount(string name, decimal balance)
+        {
+            _name = name;
+            _balance = balance;
+        }
+
+        public string ShowUserNameAndBalance()
+        {
+            if(_balance < 0)
+            {
+                _balance *= -1;
```

Comment:
Kāpēc reizini un tad tik un tā rādi ar -?

Reply:
Itkā daru tā tapēc jo tai mīnus zīmei jābūt pirms $ zīmes, tapec uztaisnu to skaitlu ka pozitivu skaitli un manuali pielieku to - zīmi.